### PR TITLE
Update Chakra status

### PR DESCRIPTION
**Pull request by bterlson**
Also updates ES6 links to official spec.

### CODE REVIEW

File: app/static/ie-status.json [hunk 1]
```diff
@@ -3125,10 +3125,10 @@
     "name": "Subclassing (ES6)",
     "category": "JavaScript",
     "summary": "Allows subclassing of built-in types such as Dates and Arrays.",
-    "link": "https://people.mozilla.org/~jorendorff/es6-draft.html",
+    "link": "http://ecma-international.org/ecma-262/6.0/index.html",
     "standardStatus": "Editor's Draft",
     "ieStatus": {
-      "text": "Under Consideration",
+      "text": "Under Development",
```
Comment (jacobrossi) [inline app/static/ie-status.json]: should be "In Development"

File: app/static/ie-status.json [hunk 2]
```diff
@@ -2934,7 +2934,7 @@
     "name": "Number Built-ins (ES6)",
     "category": "JavaScript",
     "summary": "New methods and properties on Number, including isFinite, isInteger, isSafeInteger, isNaN, EPSILON, MIN_SAFE_INTEGER, and MAX_SAFE_INTEGER.",
-    "link": "https://people.mozilla.org/~jorendorff/es6-draft.html",
+    "link": "http://ecma-international.org/ecma-262/6.0/index.html",
```
Comment (jacobrossi) [inline app/static/ie-status.json:2937]: Is there a hash nav we can include to jump to the specific feature (rinse and repeat this for the lines below)?

File: app/static/ie-status.json (hunk 1 above)
Comment (bterlson) [inline app/static/ie-status.json]: Fixed.

File: app/static/ie-status.json (hunk 2 above)
Comment (bterlson) [inline app/static/ie-status.json:2937]: Perhaps in some cases, but really these features span many spec sections so it's hard to link to a specific one. I'd rather handle this as part of a separate PR.

**Pull request merged**